docs/index.md

```diff
@@ -70,12 +70,13 @@ Note that LSI will work without these libraries, but as soon as they are install
 
 * Install [LAPACKE](https://www.netlib.org/lapack/lapacke.html)
   * Ubuntu: `apt-get install liblapacke-dev`
-  * macOS: (Help wanted to verify installation steps) https://stackoverflow.com/questions/38114201/installing-lapack-and-blas-libraries-for-c-on-mac-os
+  * macOS: `brew install lapack`
 * Install [OpenBLAS](https://www.openblas.net/)
   * Ubuntu: `apt-get install libopenblas-dev`
-  * macOS: (Help wanted to verify installation steps) https://stackoverflow.com/questions/38114201/installing-lapack-and-blas-libraries-for-c-on-mac-os
-* Install the [Numo::NArray](https://ruby-numo.github.io/narray/) and [Numo::Linalg](https://ruby-numo.github.io/linalg/) gems
-  * `gem install numo-narray numo-linalg`
+  * macOS: `brew install openblas`
+* Install the [Numo::NArray](https://ruby-numo.github.io/narray/) and [Numo::Linalg](https://ruby-numo.github.io/linalg/) gems. If you're using Bundler, add `numo-narray` and `numo-linalg` to your Gemfile.
```

Comment:
does the numo-linalg gem pick up the homebrew paths when built from bundler?

Reply:
No, it doesn't. If the gem is already installed correctly (using the notes below), simply adding these to the Gemfile will be fine because Bundler will use the installed Gem. But if installing for the first time, Bundler will need the build args.

If using Bundler, something like the following might be best:
```
bundle config set --global build.numo-linalg --with-openblas-dir=$(brew --prefix openblas) --with-lapack-lib="$(brew --prefix lapack)/lib"
```

I'll add a note about this.